interface/ceed-basis.c

```diff
@@ -1551,7 +1551,7 @@ int CeedBasisApplyAtPoints(CeedBasis basis, CeedInt num_points, CeedTransposeMod
     CeedCall(CeedMatrixMatrixMultiply(basis->ceed, chebyshev_coeffs_1d, interp_1d, chebyshev_interp_1d, Q_1d, P_1d, Q_1d));
```

Comment:
GitHub won't let me comment on the actual lines (1547-1548), but shouldn't `chebyshev_interp_1d` and `chebyshev_grad_1d` both be size `Q_1d * P_1d`?

Reply:
I think I got it.

Reply:
I need to think if there is a way for memcheck or some bounds checking to catch this.